spec/support/file_upload_helpers.rb

```diff
@@ -7,4 +7,12 @@ def self.upload_file(fixture:, content_type: 'text/csv')
       Rack::Test::UploadedFile.new(path, content_type)
     end
   end
+
+  module System
+    def upload_file(fixture)
+      visit(root_path)
+      path = Rails.root.join('spec', 'fixtures', 'files', fixture)
+      page.attach_file('csv_upload_form[file]', path, visible: false)
```

Comment:
Just a small thing, as you decided this is the helper which mean can be use for another upload_file field, what do you think if we move the `csv_upload_form[file]` to be a `file_input_field` param, 

```suggestion
      page.attach_file(file_input_field, path, visible: false)
```

Reply:
My preferences when it comes to abstraction is: it's better to late than to be early. So I tend to sit on these abstraction opportunity for quite a while 😅 .

But hey, this case seems like a reasonable one. Fixed in 2eb44647871ac2d4d6e3a00a514728c2bb9d84c5